### ISSUE TITLE
Fetching the NBT of a player returns a blank object

### ISSUE THREAD

**Issue by voximity** [ShaneBeee/Sk-NBeeT#8]
No matter what situation, fetching the NBT of any player will return a blank object, that is "{}".

Here are some instances:
```
command /mynbt:
    trigger:
        send the player's nbt to the player
```
Will output `{}` to the player's chat. No need to format as a text as it already is one.

```
on death of player:
    set {_xp} to tag "XpTotal" of the player's nbt parsed as a number
    spawn {_xp} of xp at the player
```
Won't do anything. Again, the NBT here is `{}`.

I am running on 1.14.4, Paper 156, latest Skript, 2.4.3 SK-NBeeT.

**Comment by ShaneBeee** [comment]
Yeah, I noticed that a while back. Since a player is an entity, it should be able to pull that NBT data from a player, but for some reason it's not. 

Im not sure if Im just using the wrong method, or its just not possible to do, but I will mark this as a bug and see what I can do to fix it.

**Comment by ShaneBeee** [comment]
This has been fixed in the latest release, 2.4.4
